### ISSUE TITLE
Disable the right click context menu from the webpage.

### ISSUE THREAD

**Issue by Creatoon** [Creatoon/Jar-web-app#15]
**Is your feature request related to a problem? Please describe.**
Right now the anyone can see all the messages and IDs (Document id in MongoDB of that message) and all the precious information related to a certain message just by inspecting.

**Describe the solution you'd like**
So what we want is to remove the right-click context menu, so that a normal person can not see that many details.

**Comment by payaldutta** [comment]
I want to work on this issue.
Do assign, if possible.
Thank you.
@Creatoon

**Comment by Creatoon** [comment]
@payaldutta go Ahead

**Comment by payaldutta** [comment]
@Creatoon Can you please explain to me from where should I start?
As me being a beginner will be very kind of you.
Thank You.

**Comment by harshdeepkanwar** [comment]
Hello @Creatoon , I have read this issue well and I want to work on this.
Thank You.

**Comment by Creatoon** [comment]
@harshdeepkanwar as @payaldutta said to work first, lets give her 2 days time, after you can take it.

**Comment by Creatoon** [comment]
@payaldutta are you able to run the application on your localhost

**Comment by harshdeep2121** [comment]
> @harshdeepkanwar as @payaldutta said to work first, lets give her 2 days time, after you can take it.

Ok

**Comment by payaldutta** [comment]
@Creatoon Can you please guide me with the steps to run the application on my local desktop?

**Comment by payaldutta** [comment]
@Creatoon yes, please guide in which file should I do the required change.

**Comment by Creatoon** [comment]
@payaldutta please try to explore yourself first, ask the right question, don't ask questions like which file I have to change, as this learning, if you will not able to solve, ping me, I will help you.

**Comment by payaldutta** [comment]
@Creatoon I am so sorry if I had asked something irrevalent to the issue. Thanks for cooperating, I appreciate it. I just wanted to ask about the location of HTML. index page where the coding to be done for required changes as I am unable to locate it.
Thank You.

**Comment by Creatoon** [comment]
@payaldutta I have sent you the gitter  channel request for our community, please join, further doubts about this issue will be solved there.

**Comment by harshdeepkanwar** [comment]
Hello @Creatoon , Can I try now ?
